.github/workflows/regression-tests.yml

```diff
@@ -41,3 +42,10 @@ jobs:
         cd regression-tests && ^
         bash run-tests.sh -c ${{ matrix.compiler }}
       shell: cmd
+      continue-on-error: true
+
+    - name: Upload patch
+      uses: actions/upload-artifact@v4
+      with:
+        name: ${{ matrix.compiler }}-patch.diff
```

Comment:
How about:
```suggestion
        name: ${{ matrix.os }}-${{ matrix.compiler }}-patch.diff
```
I think we might have the same compiler run on different OSes (I have that in cooking). Comparing patches for the same compiler between different systems could be another check, also of the CI. Thoughts?

BTW do you what is the retention time of artefacts on GitHub is? I guess these will be used instantly or not at all so it probably isn't an issue...

Reply:
From <https://github.com/actions/upload-artifact#retention-period>:
> Artifacts are retained for 90 days by default.